whatpulse_fr.ts

```diff
@@ -1637,7 +1637,8 @@ Please check your permissions on: %2</source>
     <message>
         <location filename="../interface/ExportWindow.cpp" line="73"/>
         <source>Uptime</source>
-        <translation>Temps de fonctionnement</translation>
+        <translatorcomment>Switch back to &quot;Uptime&quot; because &quot;Temps de fonctionnement&quot; didn&apos;t fit and it looked ugly</translatorcomment>
```

Comment:
Hey, thanks for all this! 2 Questions about this one; does `Uptime` work for [only] French-speaking people? I.e., would they know what it means? And if it's changed here, would it be better to change it consistently?

Asking because I could also change the UI in this case to make it fit better.

Reply:
I'd say `uptime` is part of the "computer slang" that's kind of widely known amongst French-speaking people, and I use it all the time.

But to make sure I asked quite a few random people if they knew the word, and so far I've collected 77 answers. 
> ~40% know exactly what it means, ~50% do not know and ~10% are unsure.

So I would say that making this change consistent wouldn't probably be a good idea, but only changing the tab's title shouldn't be too harmful because people that do not know what `Uptime` is would instantly understand it once they go to that tab.

I know inconsistency is an issue, and I tried searching for synonyms or shorter ways to write it, but only came up with `Tps. de fonctionnement`, which is the abridged version, except it's not easy to read and is still far too long.

But if you do not mind changing the UI to make it fit better, it could as well be a good alternative!
Sorry for the inconvenience.

Reply:
I see what you mean now, I think you're talking about the main tabs and not the export window string that you changed. This one, right?

![2022-07-05_13-27-35](https://user-images.githubusercontent.com/6500836/177317513-78ac04ee-1f4c-4dd1-bb66-3e35bfe1a165.png)
 
If yes, what about using `Temps` and just say "time" ? That would still keep it French and time still relates to the features behind the tab.